src/index.js

```diff
@@ -460,7 +460,7 @@ export default class Critters {
             // Strip pseudo-elements and pseudo-classes, since we only care that their associated elements exist.
             // This means any selector for a pseudo-element or having a pseudo-class will be inlined if the rest of the selector matches.
             if (sel !== ':root') {
-              sel = sel.replace(/(?:>\s*)?::?[a-z-]+\s*(\{|$)/gi, '$1').trim();
+              sel = sel.replace(/(\\?)::?[a-z-]+(\(.+?\))?\s*/gi, (match, p1) => p1 === '\\' ? match : ' ').trim();
```

Comment:
Here's a few more cases that could be handled:
- `bar:not(.foo)` - we delete `:not(.foo)`, which is not a pseudo-selector and should be preserved/used
- `bar:not(:hover)` - deleting `:hover` produces `bar:not()`, which would be invalid
- `bar:not(:not(.foo))` - not safe to remove this

The following lookahead/lookbehind version should handle them:
```suggestion
              sel = sel.replace(/(?<!\\)::?[a-z-]+(?![a-z-(])/gi, '').replace(/::?not\(\s*\)/g, '').trim();
```

Reply:
Indeed, this is better.